### PR TITLE
Fix toolbar missing logo

### LINKED ISSUE #1754: Toolbar is missing Toggl logo
The title says it all.

### PR DESCRIPTION
**Pull request by daividssilverio**
Closes #1754 

The only activities I found on the design that had the logo were the MainActivity and SettingsActivity.
There is no Activity yet for the Reports View, right?

Ok to squash.

### CODE REVIEW

File: Toggl.Giskard/Activities/SettingsActivity.cs [hunk 1]
```diff
@@ -28,8 +28,9 @@ protected override void OnCreate(Bundle bundle)
             var toolbar = FindViewById<Toolbar>(Resource.Id.Toolbar);
 
             SetSupportActionBar(toolbar);
-            SupportActionBar.SetDisplayHomeAsUpEnabled(true);
-            SupportActionBar.SetDisplayShowHomeEnabled(true);
+            SupportActionBar.SetDisplayHomeAsUpEnabled(false);
+            SupportActionBar.SetDisplayShowHomeEnabled(false);
+            SupportActionBar.SetDisplayShowTitleEnabled(false);
 
             toolbar.NavigationClick += navigationClick;
```
Comment (willsb) [inline Toggl.Giskard/Activities/SettingsActivity.cs]: Is this even needed now that home as up enabled is set to false?

Comment (daividssilverio) [reply Toggl.Giskard/Activities/SettingsActivity.cs]: I'll check that. If I do any changes, the PR needs to be re-reviewed, right?

**Comment by willsb** [comment]
Precisely

File: Toggl.Giskard/Activities/SettingsActivity.cs (hunk 1 above)
Comment (willsb) [reply Toggl.Giskard/Activities/SettingsActivity.cs]: Precisely

Comment (daividssilverio) [reply Toggl.Giskard/Activities/SettingsActivity.cs]: Oops, I guess you were talking about the navigationClick right... Yeah, it's not needed anymore.
Neither is the SupportActionBar.SetDisplayHomeAsUpEnabled(false).

**Pull request merged**